source/administration/mobile-changelog.md

```diff
@@ -8,6 +8,47 @@ Latest Mattermost Mobile Apps releases:
 - [1.42.0 Release](#id18)
```

Comment:
@cwarnermm Do you know how to fix the mini-TOC to add the new release to the list?

Reply:
@amyblais This source file will present challenges because the page contains many duplicated headings, and this results in a link formed as ``#id##`` (Sphinx will auto-assign identifiers for headings when multiple headings exist). I've been addressing these by identifying the assigned link via a build preview first, then updating the source file with the link identifier assigned to that particular heading.

Reply:
When I view the latest preview, I'm seeing ``#release`` as the link identifier for the 1.45.0 Release. When I was adding the mini-toc initially, I noticed that some link identifiers don't follow a prescribed pattern.